### ISSUE TITLE
OS Request FluiddPI

### ISSUE THREAD

**Issue by DarkMagicSource** [raspberrypi/rpi-imager#378]
Similar to Mainsail and the varients of OctoPrint, [Fluidd](https://docs.fluidd.xyz/) is a different 3D Printer control software.

**Comment by tdewey-rpi** [comment]
Please fill out the form to request an addition to the OS list at https://forms.gle/cbsU27aLLVaxUXbw8.

**Comment by jedahan** [comment]
I was going to submit this to the form, but was wondering if someone already submitted it - don't want to submit duplicates, so it would be nice to know what has been requested, and what hasn't.

**Comment by tdewey-rpi** [comment]
We'll filter out duplicates or existing items as we process the responses.

**Comment by jedahan** [comment]
Full transparency - I’d rather not go through the work of submitting if someone else already has and it’s in your queue. Is there a way to make the form responses read-only visible? Heck even just a column for the name of the submitted distros.

**Comment by jedahan** [comment]
(If this is intentional as a filter for “is there enough interest for us to spend the work to maintain this district list”, ignore my request)

**Comment by tdewey-rpi** [comment]
Unfortunately I can't open up the results, as part of the form asks for personally identifiable information, which is collected to help coordinate future Imager changes.

That's also why the form is now a required part of adding new items to the OS list. The device filtering changes are still incomplete because we didn't have a means to contact individuals capable of making the corresponding 2P changes.

In short, without a submission we won't be able to add your OS to the list.

**Comment by jedahan** [comment]
Appreciate your patience explaining things.

**Comment by lurch** [comment]
@jedahan And just to clarify, I believe the form is intended to be used by the developers / maintainers of a particular OS image, not by 3rd-party users requesting that their favourite OS gets added to RPi Imager :wink:

**Comment by jedahan** [comment]
and fluiddpi is archived, so no need in this case unless someone revives it
